x-pack/plugins/security_solution/public/exceptions/hooks/use_import_exception_list/index.tsx

```diff
@@ -34,6 +34,7 @@ export const importExceptionList = async ({
   formData.append('file', file as Blob);
 
   const res = await http.post<ImportExceptionsResponseSchema>(`${EXCEPTION_LIST_URL}/_import`, {
+    version: '2023-10-31',
```

Comment:
I think it's worth mentioning in the description that this PR also addresses an issue with a missed version header.